### ISSUE TITLE
Add request IDs, interactive mode and user identification to Sentry context

### ISSUE THREAD

**Issue by jsierles** [superfly/flyctl#820]
Adding these to Sentry will help us trace issues across projects and group by user/mode.

**Comment by redjonzaci** [comment]
Still needed?

**Comment by jsierles** [comment]
Yes, user identification in Sentry would be helpful.

**Comment by redjonzaci** [comment]
I will get to it in a bit, but would you be able to answer any questions that may come up?

**Comment by redjonzaci** [comment]
If I understand this correctly, "request IDs, interactive mode and user identification" need to be added everywhere `sentry.WithContexts` is used?

**Comment by jsierles** [comment]
Sure thing!

**Comment by redjonzaci** [comment]
Not sure if this was a reply to my last question,
but can we get a little more specific on what attributes need to be added?

**Comment by jsierles** [comment]
Yes, we'd want to add additional context everywhere we send errors to Sentry. I'd say in order of priority, we want: user ID, request ID, interactive mode. But, the latter is the lowest hanging fruit.

**Comment by redjonzaci** [comment]
Does user ID refer to the user of an organization using `flyctl`?

**Comment by redjonzaci** [comment]
Btw I am still learning the "Ways of Working" here,
so if I could change something in communication to be more helpful,
to collaborate more efficiently, let me know!
